_source/_posts/2019-03-08-simple-rest-api-php.md

```diff
@@ -610,62 +614,46 @@ The client sends the access token to the REST API server;
 
-You can see the Issuer URI of my test Okta account in the screenshot above. Copy your own value and put it in your `.env` file.
-
-Next click the **Edit** icon, go to the **Scopes** tab and click **Add Scope** to add a scope for the REST API. We'll title it `person_api`:
-
-{% img blog/php-rest-api/create-the-person-scope.png alt:"Create the person_api scope" width:"800" %}{: .center-image }
-
-We need to add the scope to our `.env` file as well. We'll add the following to `.env.example`:
+Add to `.env.example`:
 
 ```bash
-SCOPE=
+OKTAISSUER=
+OKTACLIENTID=
+OKTASECRET=
 ```
 
-and the key with the value to .env:
+Add these keys and values to `.env`:
 
 ```bash
-SCOPE=person_api
+OKTAISSUER=https://{yourOktaDomain}/oauth2/default
+OKTACLIENTID={yourClientId}
+OKTASECRET={yourClientSecret}
 ```
 
-The next step is to create a client. Navigate to **Applications**, then click **Add Application**. Select **Service**, then click **Next**. Enter a name for your service, (e.g. People Manager), then click **Done**. This will take you to a page that has your client credentials:
-
-{% img blog/php-rest-api/view-client-credentials.png alt:"View the client credentials" width:"800" %}{: .center-image }
-
-These are the credentials that your client application will need in order to authenticate. For this example, the client and server code will be in the same repository, so we will add these credentials to our `.env` file as well (make sure to replace `{yourClientId}` and `{yourClientSecret}` with the values from this page):
+Log in to the Okta Admin Console (tip: run `okta login`, open URL in a browser). Navigate to **Security** > **API**. Select your **default** Authorization Server. Click the **Edit** icon, go to the **Scopes** tab and click **Add Scope** to add a scope for the REST API. Name it `person_api` and check **Set as a default scope**.
 
-Add to .env.example:
+Add the scope to `.env.example`:
 
 ```bash
-OKTACLIENTID=
-OKTASECRET=
+SCOPE=
 ```
 
-Add to .env:
+and the key with the value to `.env`:
 
 ```bash
-OKTACLIENTID={yourClientId}
-OKTASECRET={yourClientSecret}
+SCOPE=person_api
 ```
 
 ## Add Authentication to Your PHP REST API
 
-We'll use the Okta JWT Verifier library. It requires a JWT library (we'll use `spomky-labs/jose`) and a PSR-7 compliant library (we'll use `guzzlehttp/psr7`). We'll install everything through composer:
+We'll use the Okta JWT Verifier library. It requires a JWT library (we'll use `firebase/php-jwt`) and a PSR-7 compliant library (we'll use `guzzlehttp/psr7`). We'll install everything through composer:
 
 ```bash
-composer require okta/jwt-verifier spomky-labs/jose guzzlehttp/psr7
+composer require okta/jwt-verifier firebase/php-jwt guzzlehttp/psr7
```

Comment:
Is it possible to pin the versions for these libraries like you can with `npm install`?

Reply:
Yep, just pushed a change for that.